src/main/java/bowling/controller/BowlingGameController.java

```diff
@@ -20,36 +17,17 @@ public BowlingGameController() {
     public BowlingGameResponse startGame(BowlingGameRequest request) {
         Participant participant = new Participant(request.getParticipantName());
         service.startGame(participant);
-        return assembleGameResponse(participant, service.findFrames(participant));
+        return assembleGameResponse(service.findBowlingGame());
```

Comment:
```suggestion
        BowlingGame bowlingGame = service.findBowlingGame();
        return bowlingGame.assembleGameResponse();
```

이런식의 리뷰를 드릴려고 했으나.. 컨트롤러의 ResponseDTO를 도메인에서 사용하길 꺼려하시는 이유로 이렇게 만드신거죠? 😄 
도메인에서 컨트롤러의 ResponseDTO로 만들어서 반환하는 것이 안티패턴일까요? 🤔 
DTO가 레이어를 넘나드는 값이라고 생각 한다면 큰 문제가 없지만, 지금 구현해주신 FE - BE 구조에서는 어색한 느낌도 드네요.